src/network.zig

```diff
@@ -976,8 +976,9 @@ pub const Protocols = struct {
 		const UpdateType = enum(u8) {
 			gamemode = 0,
 			teleport = 1,
-			worldEditPos = 2,
-			timeAndBiome = 3,
+			worldEditPosToClient = 2,
+			worldEditPosToServer = 3,
```

Comment:
Why not merge them?

If you have seem separate, you'd have to check if they are on the right side anyways, otherwise this could be exploited to annoy the server host.